mne/gui/_file_traits.py

```diff
@@ -338,72 +349,49 @@ def _get_inst_fname(self):
             return '-'
 
     @cached_property
-    def _get_inst_points(self):
-        if not self.inst:
+    def _get__hsp_points(self):
```

Comment:
why 2 _ ?

Reply:
Because `_hsp_pionts` is private – `_get_` + `_hsp_points`

Reply:
(It's a method name that the HasPrivateTraits superclass recognizes)